frontend/src/metabase/visualizations/components/settings/ChartSettingFieldPicker.jsx

```diff
@@ -87,7 +87,6 @@ export const ChartSettingFieldPicker = ({
       align="center"
     >
       <ChartSettingSelect
-        data-testid="chartsettings-field-picker-select"
```

Comment:
this was just extra considering we have the `"chartsettings-field-picker"` data-testid above